City-of-Pori/pori-kada
# PORI-284 Added additional information to news as well

To test:

1. drush fra -y && drush cc all
2. Go to edit any news item. There should be "Additional information" tab. Fill something to all of the fields there.
3. View the news item. There should be additional information on the right bar. Also the lead paragraph should be bold and have some margin (didn't have that before) and the main image should be full width unlike before.

NOTICE: The contact person is visible only to logged in user. There is a separate story about solving this issue: https://wunder.atlassian.net/browse/PORI-293
